.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: clippy
-        args: --workspace --all-features -- -D warnings
+        args: --workspace --all-features --lib --bins --tests --examples -- -D warnings
```

Comment:
`--all-targets` would do the same if I understand correctly what we assume about these options (except for `--benches` but you could just remove them as part of the PR since we decided to move forward with this or just leave them since you seem to have fixed them).

Reply:
benches removed and `--lib --bins --tests --examples` replaced by `-all-targets`